python/pyspark/pandas/_typing.py

```diff
@@ -21,6 +21,12 @@
 import numpy as np
 from pandas.api.extensions import ExtensionDtype
 
+from pyspark.sql.column import Column as LegacyColumn
```

Comment:
Can you use the same naming? You call some `LegacyColumn` others `PySparkColumn`

Reply:
Sure. Let me consolidate them with `LegacyColumn`.

Reply:
Actually I prefer `PySparkXxx` to `LegacyXxx`.